qtopconf.yaml

```diff
@@ -129,9 +139,9 @@ filtering:
 
 # what parts of qtop to display
 user_display_parts:
-# - job_accounting_summary
+ - job_accounting_summary
  - workernodes_matrix
-# - user_accounts_pool_mappings
+ - user_accounts_pool_mappings
```

Comment:
next feature request: 
* support reading `getent passwd` output (cached in a file) to show info per user in the right area